src/components/Patient/PatientHome.tsx

```diff
@@ -501,6 +501,7 @@ export const PatientHome = (props: {
                         className="w-full bg-white font-semibold text-green-800 hover:bg-secondary-200"
                         id="upload-patient-files"
                         size="large"
+                        disabled={!patientData.is_active}
```

Comment:
While looking at #9294, realized that `is_active` check alone won't be enough here (since you could have a discharged patient that was re-admitted to another facility).

Refer to the changes in the aforementioned PR for getting the accurate discharged status.

Reply:
we should probably make a utility function for this by now. 😅

Reply:
Yea 😅

Reply:
@Mahendar0701 sync with @JavidSumra and create a utility function that can be imported into both files. Ensure you both are committing the same change (as far as utility fn goes) 👍

Reply:
> While looking at #9294, realized that `is_active` check alone won't be enough here (since you could have a discharged patient that was re-admitted to another facility).
> 
> Refer to the changes in the aforementioned PR for getting the live/accurate discharged status.

Done, please check the changes

Reply:
Check the comment right above ☝️ Sync with Javid to create a common fn that could be imported into both files.

Reply:
Okok , will sync with @JavidSumra